tests/components/airthings_ble/test_sensor.py

```diff
@@ -208,6 +206,7 @@ async def test_migration_with_all_unique_ids(hass: HomeAssistant):
 
     assert len(hass.states.async_all()) > 0
 
-    assert entity_registry.async_get(v1.entity_id).unique_id == v1.unique_id
-    assert entity_registry.async_get(v2.entity_id).unique_id == v2.unique_id
-    assert entity_registry.async_get(v3.entity_id).unique_id == v3.unique_id
+    # No migration should happen, unique id should be the same as before
+    assert entity_registry.async_get(v1.entity_id).unique_id == VOC_V1.unique_id
```

Comment:
Side note: I don't think we should use MockEntity for those constants as they are not used as entities. Just use a dataclass or a dict or similar to capture those values.